src/components/tenantSelection/index.tsx

```diff
@@ -2,7 +2,7 @@ import React, { FC, useState } from 'react';
 import { Language } from '@smg-automotive/i18n-pkg';
 import { EnrichedSessionUser } from '@smg-automotive/auth';
 
-import { TenantSelectionSelect } from './Select';
+import { TenantSelectionSelect } from './select';
```

Comment:
Should be upper case no?

Reply:
No, we now have `select/index` and select/List` that is reused in the header